test-requirements.txt

```diff
@@ -2,3 +2,6 @@ flake8==3.3.0
 flake8-pyi==17.3.0
 pytest==3.4.2
 mypy==0.590
+# This makes sure that the repo's stubs are accessible. Using MYPYPATH won't
+# work. See https://github.com/python/mypy/issues/5007 for more details.
+.
```

Comment:
Interesting, I didn't realize that this worked in a requirements.txt file.

Reply:
I prefer to keep the self install separate, as requirements are usually for "dependencies", but I suppose that is more of a stylistic nit.

Reply:
@shoyer Yeah as far as I can tell, anything you can put after `pip install` you can put as a line in a requirements file :)

@ethanhs That's fair. I mainly wanted to simplify the process of getting the tests to work. But we can definitely remove this and change the README to tell people to run both commands
```
pip install -r test-requirements.txt
pip install .
```
if that's what folks prefer :)

Reply:
Let's do the two command version, just so it's more obvious what is going on. It's pretty non-standard to put `.` in a requirements.txt file.

Reply:
Sounds good, I separated it out and updated the Travis file an README.